code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -5,21 +5,18 @@
 	if (src.notransform)
 		return
 
-	clamp_values()
 	..()
 
-	if(!stat)
-		use_power()
+	handle_robot_cell()
 
-/mob/living/silicon/robot/proc/clamp_values()
-	SetStunned(min(stunned, 30))
-	SetParalysis(min(paralysis, 30))
-	SetWeakened(min(weakened, 20))
-	sleeping = 0
-	adjustBruteLoss(0)
-	adjustToxLoss(0)
-	adjustOxyLoss(0)
-	adjustFireLoss(0)
+/mob/living/silicon/robot/proc/handle_robot_cell()

```

Comment:
this proc handles power usage and check whether the borg should be in low_power_mode.